docs/skills/matchers/rasanlu.md

```diff
@@ -139,30 +141,135 @@ The above skill would be called on any intent which has a name of `'ask-joke'`.
 An http response object which has been returned by the Rasa NLU API. This allows you to access any information from the matched intent including other entities, intents, values, etc.
 
 
-## Example Skill
+## Example Skill: weatherbot
+
+`__init__.py`:
 
 ```python
 from opsdroid.skill import Skill
-from opsdroid.matchers import match_rasanlu
+from opsdroid.matchers import match_rasanlu, match_regex
 
+import logging
 import json
+import random
+
+_LOGGER = logging.getLogger(__name__)
 
 class MySkill(Skill):
-    @match_rasanlu('restaurants')
-    async def dumpResponse(self, message):
-        print(json.dumps(message.rasanlu))
+    @match_rasanlu("ask_weather")
```

Comment:
Does this change mean this will also match things like `ask_weather_foo` and `ask_weather_bar`? Do we want that?